lib/config.js

```diff
@@ -272,6 +272,20 @@ const conf = convict({
       format: Array,
       env: 'HPKP_PIN_SHA256'
     }
+  },
+  serverCache: {
```

Comment:
We need to think about a phase roll out of this feature. I wonder if we enable caching only for `@mozilla.com` and `@restmail.net` emails first. Also we might do a percentage rollout, e.g only the 10% of requests are cached (based on the `uid`)